helix-core/src/test/java/org/apache/helix/integration/task/TestStopWorkflow.java

```diff
@@ -275,13 +281,16 @@ private void cleanupParticipants(int numNodes) {
    * A mock task class that models a short-lived task to be stopped.
    */
   private class StopTask extends MockTask {
+    boolean toStop = false;
```

Comment:
What does "toStop" mean? Is there a better name for this?

Reply:
Fixed. I changed it to _stopFlag.